assets/src/application/ui/layout/EntityCard/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import EntityPaperFrame from '../EntityPaperFrame';
+import { EntityCardProps } from './types';
+import './styles.scss';
+
+const EntityCard: React.FC<EntityCardProps> = ({ actionsMenu, details, entity, sidebar, reverse = false }) => {
+	const layout = reverse ? 'entity-card entity-card--reverse-layout' : 'entity-card';
```

Comment:
this is ~BOOM~ `BEM` :boom:

Reply:
using this: https://github.com/pocotan001/bem-classnames would make it even more 💥